interface/modules/custom_modules/oe-module-claimrev-connect/public/debug-info.php

```diff
@@ -0,0 +1,69 @@
+<?php
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="connectivity";
+    require_once "../../../../globals.php";
+
+    use OpenEMR\Modules\ClaimRevConnector\Bootstrap;
+    use OpenEMR\Modules\ClaimRevConnector\ClaimRevApi;
+    
+    class ConnectivityInfo
```

Comment:
classes should be in src directory with other classes

Reply:
done